viewer/packages/pointclouds/src/annotationTypes.ts

```diff
@@ -2,39 +2,20 @@
  * Copyright 2022 Cognite AS
  */
 
+import { StylableObject } from './styling/StylableObject';
 import { IShape } from './styling/shapes/IShape';
-import { Cylinder } from './styling/shapes/Cylinder';
 
-import * as THREE from 'three';
-import { Box } from './styling/shapes/Box';
-
-import { createInvertedRevealTransformationFromCdfTransformation } from './styling/shapes/linalg';
-
-export class CylinderPrimitive {
-  constructor(public centerA: THREE.Vector3, public centerB: THREE.Vector3, public radius: number) {}
-
-  transformToShape(): IShape {
-    return new Cylinder(
-      [this.centerA.x, this.centerA.y, this.centerA.z],
-      [this.centerB.x, this.centerB.y, this.centerB.z],
-      this.radius
-    );
-  }
-}
-
-export class BoxPrimitive {
-  constructor(public instanceMatrix: THREE.Matrix4) {}
-
-  transformToShape(): IShape {
-    const rawMatrix = { data: this.instanceMatrix.clone().transpose().toArray() };
-    const invertedMatrix = createInvertedRevealTransformationFromCdfTransformation(rawMatrix);
-    return new Box(invertedMatrix);
-  }
-}
-
-export type Geometry = CylinderPrimitive | BoxPrimitive;
+export type CdfPointCloudObjectAnnotation = {
+  annotationId: number;
+  assetId?: number;
+  region: IShape[];
+};
 
-export type BoundingVolume = {
+export type AnnotationMetadata = {
```

Comment:
I think this can be renamed to something more specific, e.g. `PointCloudObjectMapping` or something.

Reply:
Or maybe `PointCloudObjectMetadata`?

Reply:
`PointCloudObjectMetadata` it is